### PR TITLE
[RestResourceMonitoring][DBTablesMonitoring] Add "commentCount" property to indeint object

### PR DESCRIPTION
**Pull request by ashie**
It makes easy to get number of comments belongs to an incident.

https://github.com/project-hatohol/hatohol/issues/2119#issuecomment-195845731

### CODE REVIEW

File: server/test/testDBTablesMonitoring.cc [hunk 1]
```diff
@@ -2048,6 +2048,60 @@ void test_addIncidentHistory(void)
...
+  const UnifiedEventIdType unifiedEventId)
+{
+	for (size_t i = 0; i < NumTestIncidentInfo; i++) {
+		if (testIncidentInfo[i].unifiedEventId == unifiedEventId)
+			return &testIncidentInfo[i];
+	}
+	return NULL;
+}
+
+void test_incremaentIncidentCommentCount(void)
```
Comment (cosmo0920) [inline server/test/testDBTablesMonitoring.cc]: Oh, `increment` becomes `incremänt`! :-P

```diff
-incremaent ->
+increm ent
       -
```

File: server/test/testDBTablesMonitoring.cc [hunk 2]
```diff
@@ -2048,6 +2048,60 @@ void test_addIncidentHistory(void)
...
+	string statement =
+	  StringUtils::sprintf(
+	    "select * from incidents where unified_event_id=%"
+	    FMT_UNIFIED_EVENT_ID, history.unifiedEventId);
+	string expected = makeIncidentOutput(incident);
+	DBAgent &dbAgent = dbMonitoring.getDBAgent();
+	assertDBContent(&dbAgent, statement, expected);
+}
+
+void test_notIncremaentIncidentCommentCount(void)
```
Comment (cosmo0920) [inline server/test/testDBTablesMonitoring.cc]: Ditto.

File: server/test/testDBTablesMonitoring.cc [hunk 3]
```diff
@@ -2048,6 +2048,60 @@ void test_addIncidentHistory(void)
...
 }
 
+static const IncidentInfo *findIncidentInfoWithUnifiedEventId(
+  const UnifiedEventIdType unifiedEventId)
+{
+	for (size_t i = 0; i < NumTestIncidentInfo; i++) {
+		if (testIncidentInfo[i].unifiedEventId == unifiedEventId)
+			return &testIncidentInfo[i];
+	}
+	return NULL;
```
Comment (cosmo0920) [inline server/test/testDBTablesMonitoring.cc]: In C++11, null pointer should be written in `nullptr`.

**Comment by cosmo0920** [comment]
Hmm..., `power button press` is so powerful enemy for CI green...!

**Comment by ashie** [comment]
I fixed them.

**Comment by cosmo0920** [comment]
:+1:

File: server/src/DBTablesMonitoring.cc [hunk 4]
```diff
@@ -3691,10 +3784,12 @@ static bool updateDB(
   DBAgent &dbAgent, const DBTables::Version &oldPackedVer, void *data)
 {
 	const int &oldVer = oldPackedVer.getPackedVer();
-	if (oldVer <= DBTables::Version::getPackedVer(0, 1, 1)) {
-		// add a new column "extended_info" to events
-		DBAgent::AddColumnsArg addColumnsArg(tableProfileEvents);
-		addColumnsArg.columnIndexes.push_back(IDX_EVENTS_EXTENDED_INFO);
+	if (oldVer == DBTables::Version::getPackedVer(0, 2, 0)) {
```
Comment (cosmo0920) [inline server/src/DBTablesMonitoring.cc]: memo: getPackedVer(0, 1, 1) should be removed because major version 1 migrations no longer supported.

**Comment by ashie** [comment]
Thanks!

**Pull request merged**